synapse/lib/stormtypes.py

```diff
@@ -193,10 +193,24 @@ def addLibFuncs(self):
             'now': s_common.now,
             'fromunix': self.fromunix,
             'parse': self.parse,
+            'format': self.format,
         })
 
     # TODO from other iso formats!
 
+    async def format(self, valu, format):
+        '''
+        Format a Synapse timestamp into a string value using strftime.
+        '''
```

Comment:
we should probably check isinstance(str) and use the norm() for the time type here so specifying a string date constant works as input.